torch_frame/nn/encoder/stypewise_encoder.py

```diff
@@ -47,6 +47,20 @@ def __init__(
         self.col_names_dict = col_names_dict
         self.encoder_dict = ModuleDict()
         for stype, stype_encoder in stype_encoder_dict.items():
+            if stype != stype.parent:
```

Comment:
Shall we update the doc-string of `StypeWiseFeatureEncoder` to clarify that stype in `stype_encoder_dict` needs to parent?

Reply:
Added.